src/scrabble.js

```diff
@@ -1,5 +1,126 @@
-function scrabble() {
+const isStringZero = (str) => {
+  if (str.length === 0) {
+    str += '0'
+  }
+  return str
+}
+
+const stringToArray = (word) => {
+  const letters = []
+  word
+    .toLowerCase()
+    .split('')
+    .forEach((element) => {
+      letters.push(element)
+    })
+  return letters
+}
+
+const checkWordMulti = (array) => {
+  let wordMulti = 1
+  let wordMultis = 0
+  const datasheet = require('./datasheet.js').datasheet
+  if (Object.getOwnPropertyNames(datasheet).includes(array[0])) {
+    return [array, wordMulti]
+  } else if (
+    (array[0] === '[' && array[2] === ']') ||
+    (array[0] === '{' && array[2] === '}')
+  ) {
+    return [array, wordMulti]
+  } else {
+    for (
```

Comment:
the for loop code could have been cleaner by making the condition and things inside the bracket all in one line